### ISSUE TITLE
[BUG] Read more do not have aria-label

### ISSUE THREAD

**Issue by YuriDevAT** [YuriDevAT/accessibility-first-issue-tracker#13]
### Description

The read more button link does not have additional information for screen reader users to announce where this link is leading.
